src/main/java/org/folio/rest/impl/SubjectSourceApi.java

```diff
@@ -0,0 +1,73 @@
+package org.folio.rest.impl;
+
+import static io.vertx.core.Future.succeededFuture;
+import static org.folio.rest.support.EndpointFailureHandler.handleFailure;
+
+import io.vertx.core.AsyncResult;
+import io.vertx.core.Context;
+import io.vertx.core.Handler;
+import java.util.Map;
+import javax.ws.rs.core.Response;
+import org.folio.rest.jaxrs.model.SubjectSource;
+import org.folio.rest.jaxrs.resource.SubjectSources;
+import org.folio.services.subjectsource.SubjectSourceService;
+
+public class SubjectSourceApi implements SubjectSources {
```

Comment:
@JavokhirAbdullayev  The same about documentation.

Reply:
regarding to this, its interface already contains docs with examples